### DISCUSSION TITLE
Need help, why the component plugin can not update realtime in screen during development?

### DISCUSSION THREAD

**Discussion by yoyogis** [Budibase/budibase#14583] [Help]
I'm new to Budibase. Today, I used the Budibase CLI to start a DEV environment and created a new component plugin. However, when I change the code in "/src/Component.svelte", the component in "Design/Screen" isn't updated. Only after I delete the dist directory and restart 'yarn watch', then refresh the builder page, will the component be updated.

<img width="2351" alt="image" src="https://github.com/user-attachments/assets/0086def3-3746-4fe7-b7fe-7abba5f0991d">

**Comment by mikesealey** [comment]
Hey @yoyogis does the directory you've run `yarn watch` in match the location of the component?
